src/app/shared/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -0,0 +1,13 @@
+<breadcrumb #parent>
+    <ul class="breadcrumb">
+        <ng-template ngFor let-route [ngForOf]="parent.breadcrumbs">
+            <li *ngIf="!route.terminal" class="breadcrumb-item">
```

Comment:
instead of using 2 if statements consider usage of ng-container + *ngTemplateOutlet